src/store/actions/onboardingActions.js

```diff
@@ -15,14 +15,14 @@ export const GET_PROFILEINFO_START = "GET_PROFILEINFO_START";
 export const GET_PROFILEINFO_SUCCESS = "GET_PROFILEINFO_SUCCESS";
 export const GET_PROFILEINFO_ERROR = "GET_PROFILEINFO_ERROR";
 
-export const postApplicantOnboarding = value => dispatch => {
+export const postApplicantOnboarding = (value) => (dispatch) => {
   dispatch({ type: APPLICANT_ONBOARDING_POST_START });
   //   return axiosWithAuth()
```

Comment:
Good idea commenting this stuff out so that once backend is connected, hooking up will be quick!